src/main/webapp/static/js/controller/pokemon_controller.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+angular.module('myApp').controller('PokemonController', ['$scope', 'PokemonService', function ($scope, PokemonService) {
+    var self = this;
+    self.pokemon = { id: null, name: '', attack: '', defense: '' };
+    self.pokemons = [];
+
+    self.submit = submit;
+    self.edit = edit;
+    self.remove = remove;
+    self.reset = reset;
+
+
+    fetchAllPokemons();
+
+    function fetchAllPokemons() {
+        PokemonService.fetchAllPokemons()
+            .then(
+                function (d) {
+                    self.pokemons = d;
+                    console.log('fetchAllPokemons(): self.pokemons ', self.pokemons);
```

Comment:
inject $log  into your Service and Controllers, and do not use the default console.log()